web-server/src/utils/logFormatter.ts

```diff
@@ -0,0 +1,95 @@
+import { LogLevel } from '@/types/resources';
+
+import {
+  ParsedLog,
+  generalLogRegex,
+  httpLogRegex,
+  redisLogRegex,
+  postgresLogRegex,
+  dataSyncLogRegex
+} from '../constants/log-formatter';
+
+export const parseLogLine = (rawLogLine: string): ParsedLog | null => {
+  const generalLogMatch = rawLogLine.match(generalLogRegex);
+  if (generalLogMatch) {
+    const [_fullLog, timestamp, _unused, logLevel, message] = generalLogMatch;
+    return {
+      timestamp,
+      logLevel,
+      message
+    };
+  }
+
+  const httpLogMatch = rawLogLine.match(httpLogRegex);
+  if (httpLogMatch) {
+    const [
+      _fullLog,
+      ip,
+      _unused,
+      _unused2,
```

Comment:
same with these :)

Reply:
changed the second one to _username , and first one with _ (underscore) [refer to this](https://docs.gunicorn.org/en/stable/settings.html#:~:text=remote%20address-,l,%27%2D%27,-u)